src/server/command_registry.cc

```diff
@@ -115,6 +116,19 @@ CommandId::CommandId(const char* name, uint32_t mask, int8_t arity, int8_t first
   implicit_acl_ = !acl_categories.has_value();
 }
 
+CommandId::CommandId(const char* name, uint32_t mask, int8_t arity, int8_t first_key,
```

Comment:
you do not need it.

Reply:
you still copy the handler function outside of the c'tor - your Clone function can fix whatever needs to be fixed anyways, there is no need for another special function imho.